lib/collections/schemas/tags.js

```diff
@@ -86,7 +86,10 @@ export const Tag = new SimpleSchema({
     }
   },
   updatedAt: {
-    type: Date
+    type: Date,
+    autoValue: function () {
+      return new Date;
```

Comment:
should this be `new Date();`?

Reply:
I don't believe so. I don't see us doing that elsewhere. e.g. in `revisions.js`
```
  updatedAt: {
    type: Date,
    autoValue: function () {
      return new Date;
    },
    optional: true
  },
```

although the [collection2 docs](https://github.com/aldeed/meteor-collection2#autovalue) show `new Date()` so I am guessing both forms are correct?